config/schemas/settings.json

```diff
@@ -378,6 +378,10 @@
 					"type" : "boolean",
 					"default" : true
 				},
+				"extraResolutionsModPath" : {
+					"type" : "string",
+					"default" : "/vcmi-extras/Mods/extraResolutions/Content/config/resolutions.json"
```

Comment:
maybe a vector of configs to allow resolutions from other mods too?

Reply:
I agree, good idea

Reply:
mod name should also be placed here then instead of hardcoding